dbfit-java/core/src/main/java/dbfit/api/DbObject.java

```diff
@@ -1,12 +1,11 @@
 package dbfit.api;
 
-import dbfit.fixture.StatementExecution;
 import dbfit.util.DbParameterAccessor;
 import dbfit.util.Direction;
 
 import java.sql.SQLException;
 
 public interface DbObject {
-    public StatementExecution buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException ;
+    public DbCommand buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException;
```

Comment:
Hmm, now I see that the method name is maybe not ideal. On the other hand we don't want go too deep here (as it may end up like #463).
It's good to look again and compare the HSQLDB changes - and see whether would be better to introduce `DbCommand` before or after it.